.github/workflows/build_php71.yaml

```diff
@@ -0,0 +1,77 @@
+name: Rector Run
+
+on:
+    pull_request: null
+    push:
+        branches:
+            - master
+
+jobs:
+    downgrade_php71:
+        runs-on: ubuntu-latest
+
+        steps:
+            -   uses: actions/checkout@v2
+
+            # see https://github.com/shivammathur/setup-php
+            -   uses: shivammathur/setup-php@v2
+                with:
+                    php-version: 7.4
+                    coverage: none
+
+            # The Symfony Polyfill libraries are needed
+            -   run: composer require symfony/polyfill-php72 symfony/polyfill-php73 symfony/polyfill-php74 symfony/polyfill-php80 --no-install --no-update
+
+            # Execute the downgrade script
+            -   run: ./.github/workflows/scripts/downgrade_packages.sh 7.1
+
+            # Upload the artifact so we can download and review the downgraded code
+            -   run: mkdir build
+            -   uses: montudor/action-zip@v0.1.1
+                with:
+                    args: zip -X -r build/rector-php71.zip . -x *.git* .* "*/\.*" build** ci** stubs** tests** "**/tests**"
+            -   uses: actions/upload-artifact@v2
+                with:
+                    name: rector-downgraded-code
+                    path: build/rector-php71.zip
+
+            # Run all the tests on PHP 7.4 (because JUnit needs PHP 7.3+),
+            # but using Rector downgraded to PHP 7.1
+            -   run: vendor/bin/phpunit
+
+            # Avoid Composer v2 platform checks (composer.json requires PHP 7.3+, but below we are running 7.1)
+            -   run: composer config platform-check false
+            # Regenerating the autoload files will remove vendor/composer/platform_check.php, otherwise executing bin/rector on 7.1 fails
+            -   run: composer dumpautoload
+
+            # Remove DEV dependencies
+            -   run: composer install --no-dev
+
+            # Replace the composer.json file
+            -   run: cp ci/downgrade/composer.json ./composer.json
```

Comment:
Override to remove dependencies, maybe put into scsoper sympfliy

Reply:
To confirm: you'd rather read the current `composer.json`, remove all entries in `require` using `ComposerJson`'s `setRequire` from Symplify, and save again?

Isn't the current solution much simpler? More since [the new `composer.json`](https://github.com/rectorphp/rector/blob/ci-build-php71/ci/downgrade/composer.json) is completely empty... (unless there's something from the current `composer.json` that we need to keep?)